src/test/java/org/folio/rest/impl/OaiPmhSetImplTest.java

```diff
@@ -57,6 +59,8 @@
   private static final String EXISTENT_SET_ID = "16287799-d37a-49fb-ac8c-09e9e9fcbd4d";
   private static final String NONEXISTENT_SET_ID = "a3bd69dd-d50b-4aa6-accb-c1f9abaada55";
 
+  private static final String NULL_VALUE_ERROR_MSG_TEMPLATE = "null value in column \"%s\" violates not-null constraint";
```

Comment:
Put this in a constants class so we don't have to repeat it?

Reply:
I've replied on it into https://github.com/folio-org/mod-oai-pmh/pull/128